features/step_definitions/reek_steps.rb

```diff
@@ -11,69 +11,72 @@
 end
 
 Then /^it reports nothing$/ do
-  assert_exact_output '', all_stdout
+  expect(last_command_started).to have_output_on_stdout('')
```

Comment:
Or, maybe we should do what this line does: https://github.com/cucumber/aruba/blob/bfb67969567cf9d74140c1957e58cf65251adda7/lib/aruba/cucumber/command.rb#L312